src/admin-views/attendees.php

```diff
@@ -28,9 +29,9 @@
 			<?php
 			echo esc_html(
 				sprintf(
-				// Translators: 1: the post title, 2: the post ID.
+					// Translators: %1$s: the post title, %2$d: the post ID.
```

Comment:
The same comment here about context for translators and maybe including `post/event`

Reply:
```suggestion
					// Translators: %1$s: the post/event title, %2$d: the post/event ID.
```